yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -801,33 +801,65 @@ func addUnsupportedFeaturesFromSchemaAnalysisReport(featureName string, issueRea
+	unsupportedFeatures = append(unsupportedFeatures, getUnsupportedFeaturesFromSchemaAnalysisReport("Storage parameters in DDLs", STORAGE_PARAMETERS_DDL_STMT_ISSUE, schemaAnalysisReport, true, ""))
+	unsupportedFeatures = append(unsupportedFeatures, getUnsupportedFeaturesFromSchemaAnalysisReport("Extensions", UNSUPPORTED_EXTENSION_ISSUE, schemaAnalysisReport, false, ""))
+	unsupportedFeatures = append(unsupportedFeatures, getUnsupportedFeaturesFromSchemaAnalysisReport("Exclusion constraints", EXCLUSION_CONSTRAINT_ISSUE, schemaAnalysisReport, false, ""))
+	unsupportedFeatures = append(unsupportedFeatures, getUnsupportedFeaturesFromSchemaAnalysisReport("Deferrable constraints", DEFERRABLE_CONSTRAINT_ISSUE, schemaAnalysisReport, false, ""))
+	unsupportedFeatures = append(unsupportedFeatures, getUnsupportedFeaturesFromSchemaAnalysisReport("View with check option", VIEW_CHECK_OPTION_ISSUE, schemaAnalysisReport, false, ""))
+	unsupportedFeatures = append(unsupportedFeatures, getIndexesOnComplexTypeUnsupportedFeature(schemaAnalysisReport))
 	return unsupportedFeatures, nil
 }
 
+func getIndexesOnComplexTypeUnsupportedFeature(schemaAnalysisiReport utils.SchemaReport) UnsupportedFeature {
+	jsonbIndexes := getUnsupportedFeaturesFromSchemaAnalysisReport("JSONB", fmt.Sprintf(ISSUE_INDEX_WITH_COMPLEX_DATATYPES, "jsonb"), schemaAnalysisReport, false, "")
+	tsvectorIndexes := getUnsupportedFeaturesFromSchemaAnalysisReport("TSVECTOR", fmt.Sprintf(ISSUE_INDEX_WITH_COMPLEX_DATATYPES, "tsvector"), schemaAnalysisReport, false, "")
+	tsqueryIndexes := getUnsupportedFeaturesFromSchemaAnalysisReport("TSQUERY", fmt.Sprintf(ISSUE_INDEX_WITH_COMPLEX_DATATYPES, "tsquery"), schemaAnalysisReport, false, "")
+	citextIndexes := getUnsupportedFeaturesFromSchemaAnalysisReport("CITEXT", fmt.Sprintf(ISSUE_INDEX_WITH_COMPLEX_DATATYPES, "citext"), schemaAnalysisReport, false, "")
+	inetIndexes := getUnsupportedFeaturesFromSchemaAnalysisReport("INET", fmt.Sprintf(ISSUE_INDEX_WITH_COMPLEX_DATATYPES, "inet"), schemaAnalysisReport, false, "")
+	arrayIndexes := getUnsupportedFeaturesFromSchemaAnalysisReport("ARRAY", fmt.Sprintf(ISSUE_INDEX_WITH_COMPLEX_DATATYPES, "array"), schemaAnalysisReport, false, "")
+	indexesOnComplexTypesFeature := UnsupportedFeature{
+		FeatureName: "Index on complex datatypes",
+		DisplayDDL:  false,
+		Objects: []ObjectInfo{},
+	}
+	//append objects from each of these to indexesOnComplexTypesFeature
+	appendObjects := func(feature UnsupportedFeature) {
+		for _, object := range feature.Objects {
+			formattedObject := object
+			formattedObject.ObjectName = fmt.Sprintf("%s: %s", feature.FeatureName, object.ObjectName)
+			indexesOnComplexTypesFeature.Objects = append(indexesOnComplexTypesFeature.Objects, formattedObject)
+		}
+		indexesOnComplexTypesFeature.DocsLink = feature.DocsLink
```

Comment:
This would be a common link? You probably shouldn't be overriding the link with the link fromm each feature?

Reply:
Yes, this is a common link.

Reply:
Yes, so my point was that we don't really need to update it for each index type, then, right?

Reply:
yes fixed